elements/rh-navigation-secondary/rh-navigation-secondary.ts

```diff
@@ -6,7 +6,6 @@ import { state } from 'lit/decorators/state.js';
 import { queryAssignedElements } from 'lit/decorators/query-assigned-elements.js';
 
 import { ComposedEvent } from '@patternfly/pfe-core';
-import { RovingTabindexController } from '@patternfly/pfe-core/controllers/roving-tabindex-controller.js';
```

Comment:
Focus is lost briefly when tabbing, here's how to reproduce:

1. Visit the [DP demo for secondary nav](https://deploy-preview-1821--red-hat-design-system.netlify.app/elements/navigation-secondary/demo/).
2. Hit "Tab" on your keyboard until you get to the first dropdown.
3. Open the first dropdown.
4. Keep hitting Tab to tab through every element in the dropdown.
5. When you get to the last focusable item in the dropdown, hit tab again.
6. **Notice the dropdown disappears but the current focused item is lost.**
7. If you hit Tab one more time, you'll notice focus returns again and is on the next dropdown menu toggle.

Expected results: focus should remain visible for each tab stop.

Related to [WCAG 2.4.7 Focus Visible](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html)

If you test this on ux.redhat.com, note the difference in functionality there vs here.

Reply:
If you test this with VO + Safari, it looks like Safari returns focus to the first focusable element on the page (the logo, in this case).

Reply:
removed a `event.preventDefault()` that was still included.  RTI would reset focus given the next index after you tabbed out of a dropdown menu, so it took control, so the preventDefault was there to ensure tab didn't first set focus.  PTAL.